### DISCUSSION TITLE
PCL2联机问题

### DISCUSSION THREAD

**Discussion by minecraftcoffe** [Meloong-Git/PCL#1633] [提问] [answered]
这联机开发不用这么长时间吧。。。应该好了吧。。。

**Comment by iPodtouch6557** [comment]
可能还真需要呢.jpg
而且 #1559 提到过了，也有一些人问过了，不建议交相关问题了，这样很烦

**Comment by Chiloven945** [comment]
或许你可以试一试

**Comment by WForst-Breeze** [comment, marked as answer]
请见 #1146 #1579

> 目前 Octo Network 章鱼网络的早期开发还在进行当中，没有确切的时间表，具体请参阅各启动器开发者在交流群内发布的相关公告为准。

> 有一些许可证相关的事情，这个是耗时较长的
